controllers/profile_controller.go

```diff
@@ -72,6 +72,7 @@ func (r *ProfileSubscriptionReconciler) SetupWithManager(mgr ctrl.Manager) error
 		)). // Owns ensures that changes to resources owned by the pSub cause the pSub to get requeued
 		Owns(&sourcev1.GitRepository{}).
 		Owns(&helmv2.HelmRelease{}).
+		Owns(&sourcev1.HelmRepository{}).
```

Comment:
think this was missing from the introduction of helmrepo resources

Reply:
Ah yes, nice.